raiden/constants.py

```diff
@@ -15,6 +15,14 @@
 UINT256_MAX = 2 ** 256 - 1
 UINT64_MAX = 2 ** 64 - 1
 
+UNLIMITED_TOKEN_NETWORKS = UINT256_MAX
```

Comment:
The naming of all these 3 unlimited variables is confusing. I would drop them all and just use UINT256_MAX directly wherever you use them. That conveys the fact that there is no real limit in that case much better imo.

Alternatively, but I like it less, just call them `MAX_TOKEN_NETWORKS`, `MAX_PER_CHANNEL...` e.t.c.
Anything but "unlimited limit" :sweat_smile: